pydra/engine/tests/test_newnode.py

```diff
@@ -25,8 +27,7 @@ def move2orig():
     request.addfinalizer(move2orig)
 
 
-Plugins = ["serial"]
-Plugins = ["serial", "mp", "cf", "dask"]
+plugins = ["serial", "mp", "cf", "dask"]
```

Comment:
I like using capital letter for global var, we can always use all caps names if people prefer

Reply:
i would prefer all caps for global. title case makes it look like a class. i'll change it to all caps.

Reply:
ok